### PR TITLE
feat: seo markup vanilla

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/options/vanilla/_faq-schema.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    '@type': contentType
+    name: string
+    acceptedAnswer: AcceptedAnswerType
+}
+
+type FaqSchemaType = {
+    '@context': string
+    '@type': contentType
+    mainEntity: MainEntityType[]
+}
```
Comment (hasan-deriv) [inline src/features/pages/options/vanilla/_faq-schema.ts:20]: Other options pages also need this type. We should make this type reusable.

Comment (Nuzhy-Deriv) [reply src/features/pages/options/vanilla/_faq-schema.ts]: done. please review

**Review by hasan-deriv — APPROVED** [review]

File: src/features/types/faq-schema.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+type contentType = 'FAQPage' | 'Question' | 'Answer'
+
+type AcceptedAnswerType = {
+    '@type': contentType
+    text: string
+}
+
+type MainEntityType = {
```
Comment (mayuran-deriv) [inline src/features/types/faq-schema.ts:8]: hi man  good move can you update this in accumulated schema too, if you don't mind, please

**Review by mayuran-deriv — APPROVED** [review]

**Review by konstantinosG-deriv — APPROVED** [review]

File: src/features/types/faq-schema.ts (hunk 2 above)
Comment (Nuzhy-Deriv) [reply src/features/types/faq-schema.ts:8]: hi man, it's updated in accumulator already. plus, updated the "How do accumulator options work" tag to h2

**Pull request merged**